services/catarse/spec/models/membership/tier_spec.rb

```diff
@@ -10,6 +10,44 @@
     it { is_expected.to have_many(:subscriptions).class_name('Membership::Subscription').dependent(:destroy) }
   end
 
+  describe 'Before Validation' do
+    subject(:result) { described_class.create(project_id: project.id, attributes: attributes) }
+
+    let(:project) { create(:project) }
+
+    context 'when the order attribute is not populated and there are one or more tiers' do
+      let!(:tiers) do
+        [
+          create(:membership_tier, order: 1, project: project),
+          create(:membership_tier, order: 2, project: project),
+          create(:membership_tier, order: 3, project: project)
+        ]
+      end
+
+      let(:attributes) { attributes_for(:membership_tier).except(:order).merge(project_id: project.id) }
+
+      it 'the order attribute is the highest value plus a unit' do
+        expect(result.order).to eq(tiers.last.order + 1)
+      end
+    end
+
+    context 'when the order attribute is not populated and there are no more tier' do
+      let(:attributes) { attributes_for(:membership_tier).except(:order).merge(project_id: project.id) }
+
+      it 'the order attribute is equal to one' do
+        expect(result.order).to eq(1)
+      end
+    end
+
+    context 'when the order attribute is populated' do
+      let(:attributes) { attributes_for(:membership_tier).merge(project_id: project.id) }
+
+      it 'there is no change in the attribute order' do
+        expect(result.order).to eq(attributes[:order])
```

Comment:
**Sugestão**
Tu pode criar um nível de assinatura com uma ordem fixa, assim: `create(:membership_tier, order: 40)` e depois testar que a ordem dela é 40, assim: `expect(tier.order).to eq 40`.